docs/eventing/samples/kafka/source/index.md

```diff
@@ -7,6 +7,12 @@ type: "docs"
 
 Tutorial on how to build and deploy a `KafkaSource` [Eventing source](../../../sources/README.md) using a Knative Serving `Service`.
 
+## Background
+
+The `KafkaSource` reads all the messages, from all partitions, and converts these Kafka Records to a CloudEvent and distributes those via HTTP to its configured `sink`.
```

Comment:
Do you want to say that when the record is already a CloudEvent, there is no conversion?
Do you want to say anything about QoS? at-least-once, partition ordering is preserved

Reply:
updated the sentence a bit!